### DISCUSSION TITLE
We need collapsable notes/bullet points

### DISCUSSION THREAD

**Discussion by SpaceRaccoonFluff** [standardnotes/forum#2863] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Anyone here who takes extensive notes, which I'm assuming is many, knows how much of a PAIN it is to scroll thru your notes to find something you are looking for when you're constantly looking back and forth. **Unfortunately, the more you write the more unbearable it becomes to use Standard Notes.** 

**Describe the solution you'd like**
I want this beautiful note nesting that Workflowy has https://i.imgur.com/o6XvSQx.png. 

**Describe alternatives you've considered**
I use bullet points extensively for ALL my notes. I specfially make sure the first primary bullet point is short and concise to make it easier to find what I'm looking for when scrolling aggressively. It's still a pain to quickly find as fast as it should be. 

**Additional context**
https://i.imgur.com/o6XvSQx.png

**Comment by vstepaniuk** [comment]
Have you tried scrolling the Google Doc app? That's cool

**Comment by myreli** [comment]
The [new Super note type](https://youtu.be/bkp6nrGqjUM?t=370) offers nested collapsible blocks:

<img width="841" alt="image" src="https://user-images.githubusercontent.com/17554234/221366409-a9c07d03-05c2-4ec8-bd28-39cdead3f8c3.png">

**Reply by SpaceRaccoonFluff** [reply]
WOW am I incredibly grateful to see this actually implemented and already available to use within Standard Notes. This is going to be a huge one for upping the level of productivity I can be considering how much time I spend within my notes on a daily basis. I just got the chance to test it. And first time hearing about it too. I love it but a couple issues so far that I hope gets addressed:

1st Converting all my Rich Text notes to Super is just a total mess. Like everything is in plain text, no format, no spaces, no bullet points, I'm having a hard time reading them in plain text as well. < ul > < li > everywhere. I really hope a real conversion option will be available because lets just say my Email backups need to be divided into 4 separate emails and that level of notes needing conversion by hand is going to be a no go to the point that I would have to consider my prior notes part of some fragmented archive instead of building upon them any further. 

2nd It would be cool to be able to change the size of the Title for Collapsible blocks for easier viewing and prioritizing certain blocks over others visually as I plan to switch all my notes over into blocks eventually and being able to quickly differentiate within different blocks visually would be a nice touch.

3rd It would be nice to be able to have the collapsible blocks be able to be placed on a bullet point themselves. This would allow me to have collapsible blocks as part of within a major bullet pointed note.

Really excited to start using it. Thanks.

**Reply by myreli** [reply]
I'm glad you are enjoying it! 

Super is where the team is currently actively focused, so you should see improvements very frequently. 

1. Yes, currently the converter is from Markdown notes and Plaintext. I'll take note of this, but I'm confident that Rich Text is probably a long-term goal. 
2. Good suggestion. 
3. That's a great suggestion too, I can see myself using it.

**Reply by affvalente** [reply]
Although the collapsible block helps, it's not the same as having the native markdown h1, h2, h3 be collapsible. That would be powerful

The collapsible block breaks on export back to markdown and makes an enormous mess at it too

**Reply by FractalHabits** [reply]
This would be great. RemNote has a great one like this, where you can collapse any bullet and any bullet can also be a task. It's important to be able to create nested bullets and the current collapse feature is kinda glitchy.
